Components/card/card.css

```diff
@@ -0,0 +1,138 @@
+@import url(../Shared/reset.css);
+@import url(../Shared/colors.css);
+
+.card {
+    background: var(--light);
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+    padding: 0.5rem;
+    max-width: 23rem;
+}
+
+.card-badge,
+.card {
+    display: inline-block;
+}
+
+.card-body-horizontal,
```

Comment:
if possible have a separate class for `border-radius`

Reply:
Here border-radius is not an optional property, if I made a class for it then the styling will not get applied without this new class.